docs/automated-testing/unit-testing/README.md

```diff
@@ -179,6 +179,16 @@ Some common pitfalls when writing a unit test that are important to avoid:
   the system is not entirely testable. It is best to wrap third party API calls in interfaces or other structures so
   that they do not get invoked in unit tests.
 
+### Integration tests instead of unit tests
+
+In some situations it is worth considering to write integration tests instead of unit tests to provide full test coverage. The prerequisite for this approach to be successful is to have integration being able to execute at a speed comparable to that of unit tests both locally and in a CI environment.
```

Comment:
```suggestion
In some situations it is worth considering to write integration tests instead of unit tests to provide full test coverage. The prerequisite for this approach to be successful is to have integration tests being able to execute at a speed comparable to that of unit tests both locally and in a CI environment.
```